singularity/start.sh

```diff
@@ -6,6 +6,10 @@ set -eo pipefail
 shellmode=false
 devmode=false
 
+# Getting environment variables from .env
+source ./loadenv.sh
```

Comment:
Why are they coming above now?

Reply:
Previously, the following line always evaluated `true` because the env variables hadn't been loaded when it was evaluated:

https://github.com/PREDICT-DPACC/dpdash/blob/34b81685a433aac8618d44e92b41b45686431e73/singularity/start.sh#L19